nginx/dev/adage-nginx.conf

```diff
@@ -0,0 +1,63 @@
+server {
+    # default server - drop requests without a Host header
+    listen 80 default_server;
+    server_name "";
+    return 444;
+}
+
+# configuration of the server
+server {
+    # the port your site will be served on
+    listen      80;
+    # the domain name it will serve for
+    server_name localhost;
+    charset     utf-8;
+    access_log on;
+
+    # max upload size
+    client_max_body_size 10M;   # adjust to taste
+
+    # /static is linked to static files required for use by the adage web application
+    location /static {
+        alias /home/static;
+    }
+
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # / directs to the index.html in the static folder for the
+    # adage web application
+    # for detail on the P3P header, see http://www.w3.org/P3P/
+    location / {
+        root /home/static;
+        proxy_set_header X-Forwarded-Host $server_name;
+        proxy_set_header X-Real-IP $remote_addr;
+        add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONL UNI COM NAV"';
+    }
+
+    # Set up redirecting for all the backend endpoints we want from our django
+    # server:
+
+    location /api {
+        proxy_pass http://adage-django:8000/api;
+        proxy_set_header X-Forwarded-Host $server_name;
+        proxy_set_header X-Real-IP $remote_addr;
+        add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONL UNI COM NAV"';
+    }
+
+    location /search {
+        proxy_pass http://adage-django:8000/search;
+        proxy_set_header X-Forwarded-Host $server_name;
+        proxy_set_header X-Real-IP $remote_addr;
+        add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONL UNI COM NAV"';
+    }
+
+    location /tribe_client {
+        proxy_pass http://adage-django:8000/tribe_client;
+        proxy_set_header X-Forwarded-Host $server_name;
+        proxy_set_header X-Real-IP $remote_addr;
+        add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONL UNI COM NAV"';
+    }
```

Comment:
I am less well versed in how `tribe_client` works, so defer to you on setting it up properly. I think it would be architecturally cleaner if there were a way to ensure all of its back-end calls were channeled via `/api`, though. Is there a technical reason we have to set it up this way?

Reply:
Sure - I think that might be outside the scope of this PR, to integrate the `tribe_client` endpoints with all other `/api` endpoints. But yes, that could be a good way of doing it, and possibly other servers would benefit from it down the road, so I'll open an issue for it?